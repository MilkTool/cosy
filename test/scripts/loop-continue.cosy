print 'for min..max'
for i in 0..4 {
    if i == 2 {
        print 'skip #2'
        continue
    }
    print i
}

print 'for array'
for i in [3,4,5] {
    if i == 4 {
        print 'skip value 4'
        continue
    }
    print i
}

print 'for'
mut i = 0
for i <= 3 {
    i += 1
    if i == 2 {
        print 'skip #2'
        continue
    }
    print i
}
