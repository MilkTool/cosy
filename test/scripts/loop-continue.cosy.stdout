for min..max
0
1
skip #2
3
for array
3
skip value 4
5
for
1
skip #2
3
4
